sycl/unittests/IsCompatible/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_sycl_unittest(IsCompatibleTests OBJECT
+  #TODO: for some reason the "IsCompatible" suite fails if we place it in the
+  #binary with the "SYCL2020/SpecializationConstant". Move this to SYCL2020 when
+  #the failure is investigated.
```

Comment:
Please, file an issue to investigate the root cause of that issue and reference it here.

Reply:
Done https://github.com/intel/llvm/issues/7685

Reply:
> and reference it here

Sorry, I mean add the link to the comments in source code.